src/api/todos.ts

```diff
@@ -1,8 +1,22 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const getTodos = (userId: number) => {
+export const getTodos = (userId: number | undefined) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const addTodo = (userId: number | undefined, title: string) => {
+  return client.post<Todo>('/todos', {
+    userId,
+    completed: false,
+    title,
+  });
+};
+
+export const updatingTodo = (todoId: number, completed: boolean) => {
+  return client.patch(`/todos/${todoId}`, { completed });
+};
```

Comment:
but also you need to update title for todos
<img width="501" alt="Screenshot 2022-10-05 at 23 47 55" src="https://user-images.githubusercontent.com/47606379/194160452-662dcc52-887f-4188-9e97-8265eaeb929e.png">